src/biome/text/modules/heads/classification/classification.py

```diff
@@ -39,8 +43,19 @@ def __init__(
         self._multilabel = multilabel
 
         # metrics and loss
+        if class_weights != None:
+            if isinstance(class_weights, list):
+                class_weights = torch.FloatTensor(class_weights)
```

Comment:
Better to use:
```suggestion
                class_weights = torch.tensor(class_weights)
```
to automatically cover ints.